lua/lspconfig/server_configurations/spyglassmc_language_server.lua

```diff
@@ -4,7 +4,7 @@ return {
   default_config = {
     cmd = { 'spyglassmc-language-server', '--stdio' },
     filetypes = { 'mcfunction' },
-    root_dir = util.root_pattern 'pack.mcmeta',
+    root_dir = function() return vim.loop.cwd() end,
```

Comment:
this is not correct.

Reply:
it works though.

Reply:
it would be better to extend the default patterns to be more inclusive, maybe add a .git .svn or another specific to spyglass root directory marker, cwd() is a bad idea here.